### DISCUSSION TITLE
Buildroot and Yocto builds are broken

### DISCUSSION THREAD

**Discussion by swrpug** [Seeed-Studio/wiki-documents#632] [Q&A]
Is there a plan to update these repositories?  Neither of them build successfully.  Yocto has been broken for 2 months.  I'm disappointed with the state of these repos after spending $400 on a reterminal dm.  I was excited to see yocto and buildroot support on the wiki before I bought the dm.  It was an important reason for making the purchase.

**Comment by ricdays** [comment]
Same feeling here! I've been trying for 2 months to get a successful build! Now I was able to build the image, but the screen is black.
